mypyc/irbuild/expression.py

```diff
@@ -352,8 +352,55 @@ def transform_conditional_expr(builder: IRBuilder, expr: ConditionalExpr) -> Val
 
 
 def transform_comparison_expr(builder: IRBuilder, e: ComparisonExpr) -> Value:
-    # TODO: Don't produce an expression when used in conditional context
+    # x in (...)/[...]
+    # x not in (...)/[...]
+    if e.operators[0] in ['in', 'not in'] and isinstance(e.operands[1], (TupleExpr, ListExpr)):
```

Comment:
```suggestion
    if e.operators[0] in ['in', 'not in'] and len(e.operators) == 1 and isinstance(e.operands[1], (TupleExpr, ListExpr)):
```

Reply:
Sure, but I'd like to understand when will the number of operators in a ComparisonExpr != 1 ?

Reply:
> Sure, but I'd like to understand when will the number of operators in a ComparisonExpr > 1 ?

From `checkexpr.py`->`visit_comparison_expr`'s comment, expression like `a < b > c == d` would have multiple operators